### ISSUE TITLE
gradient not computed

### ISSUE THREAD

**Issue by antigol** [jonas-koehler/s2cnn#7]
Since the update for pytorch 0.4.0, the gradient of the layers S2Convolution and SO3Convolution are no longer computed.

model.parameters() returns all the parameters but list(model.parameters())[0].grad is None

**Comment by antigol** [comment]
status of debugging:
```grid = s2_equatorial_grid()
model = S2Convolution(1, 1, 5, 5, grid)
model.cuda()

x = torch.randn(1, 1, 10, 10).cuda()
y = model(x)
loss = y.sum()
loss.backward()

model.kernel.grad```
returns None
